openapi/components/schemas/Invoice.yaml

```
@@ -17,7 +17,7 @@ properties:
     type: integer
     x-basic: true
   orderId:
-    description: ID of the order.
+    description: ID of the order experimental.
```

suggestion: to help make this more clear
```suggestion
    description: ID of the experimental order.
```
or
```suggestion
    description: ID of the order on this experimental endpoint.
```

---

This is not an experimental endpoint, so perhaps we should apply first suggestion @noon-dawg?

---

I just so `or` in between suggestions, so updated to first one, thank you